### PR TITLE
Settings for media alias display for documents type

### PR DESCRIPTION
**Pull request by sharmasahil**
### Jira
https://digital-vic.atlassian.net/browse/SD-518
### Problem/Motivation
Media library - allow alias for media items
### Fix
Integrated new media file alias module with only document file type enabled. Also this module depends on media library from core which seems not enabled globally. Please advise if we can enable it too.

### CODE REVIEW

File: config/install/media_alias_display.settings.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+_core:
```
Comment (vincent-gao) [inline config/install/media_alias_display.settings.yml:1]: This configuration cannot reside here and you don't need it, as it is defined by another module and cannot be imported.

**Review by vincent-gao — CHANGES_REQUESTED** [review]
hey @sharmasahil, please take a look into my comments.

**Review by vincent-gao — APPROVED** [review]
Thanks @sharmasahil . It looks like a simple change. ✅

**Comment by vincent-gao** [comment]
ready for test

**Comment by sharmasahil** [comment]
> ready for test

Thanks for the approval @vincent-gao, moved it to QA, we need to still review the media_library dependency for this new module before we merge it.

**Comment by vincent-gao** [comment]
hey @sharmasahil , could you please fix the failed build

**Comment by sharmasahil** [comment]
> hey @sharmasahil , could you please fix the failed build

Fixed the build.

**Comment by sharmasahil** [comment]
> hey @sharmasahil , could you please fix the failed build

@vincent-gao Please review the lastest commit from today for the autopublish of draft media on node publish.

**Review by vincent-gao — APPROVED** [review]
LGTM, @sharmasahil 
ready for testing.

**Pull request merged**